spec/lib/task-data/schemas/install-coreos-spec.js

```diff
@@ -16,7 +16,8 @@ describe(require('path').basename(__filename), function() {
         "installDisk": "/dev/sda",
         "comport": "ttyS0",
         "osType": "linux",
-        "rootPassword": "RackHD"
+        "rootPassword": "RackHD",
+        "ignitionScriptUri": "http://172.31.128.9090/api/current/templates/ignition.json"
```

Comment:
If `ignitionScriptUri` is an optional parameter, you can add it in `positiveUnsetParam`, then the unit-test will try to test whether it is correctly set as optional. If your schema is wrongly set it as required, then the unit-test will fail.